src/util/records.py

```diff
@@ -309,6 +309,52 @@ def remove_asset(cr, name):
         remove_view(cr, name, silent=True)
 # fmt:on
 
+def get_models_inheriting_actions_actions(cr):
+    # NOTE: it would be nice to have this cached since its static info
+    # + opted for sql querying over hard-coding for better maintainability and handling custom models
+    cr.execute(
+        """
+        SELECT child_model.model
```

Comment:
Why not performing this check on the query in line 340?

Reply:
I meant that you can perform this check
https://github.com/odoo/upgrade-util/pull/81/files#diff-bdae1b00c18f37288ef023e7125a477b10e973d73dbedecc794bdb729e2daa75R336
directly in the query before, but `for_each_inherit` would also work in terms of providing a pre-computed set of models.